README.md

```diff
@@ -28,8 +28,7 @@ Run the following to build the application:
 ```shell
 git clone https://github.com/scribble-rs/scribble.rs.git
 cd scribble.rs
-go run github.com/markbates/pkger/cmd/pkger -include /resources -include /templates
-go build -o scribblers .
+make build
```

Comment:
That's a nope from me. This won't run on windows ;)

Maybe make a second section in a "show more"-tag that shows the less general solution usable on systems with make.

Reply:
Added more details there.

I'm wondering: when developing on Windows nowadays, people still use Powershell in favor of Linux Subsystem? It has been ages without Windows as a dev env for me.

Reply:
Oh, i intended it to be a new codeblock, that way the lazy bums (like me), can just copy the whole block ^^

sorry for the bickering :smile:

Reply:
Fixed, I'm used to be nitpicking king too, no worries :)